File: math/sum-of-polygon-angles.js
function sumPolygon(n) {
	return (n - 2) * 180
}

Test.assertEquals(sumPolygon(3), 180)
Test.assertEquals(sumPolygon(4), 360)
Test.assertEquals(sumPolygon(5), 540)
Test.assertEquals(sumPolygon(6), 720)
Test.assertEquals(sumPolygon(7), 900)
Test.assertEquals(sumPolygon(8), 1080)
Test.assertEquals(sumPolygon(9), 1260)
Test.assertEquals(sumPolygon(10), 1440)
Test.assertEquals(sumPolygon(11), 1620)
Test.assertEquals(sumPolygon(12), 1800)
Test.assertEquals(sumPolygon(13), 1980)
Test.assertEquals(sumPolygon(14), 2160)
Test.assertEquals(sumPolygon(15), 2340)
Test.assertEquals(sumPolygon(16), 2520)

File: math/sum-of-polygon-angles.py
def sum_polygon(n):
	return (n - 2) * 180

from time import perf_counter
tic = perf_counter()
Test.assert_equals(sum_polygon(3), 180)
Test.assert_equals(sum_polygon(4), 360)
Test.assert_equals(sum_polygon(5), 540)
Test.assert_equals(sum_polygon(6), 720)
Test.assert_equals(sum_polygon(7), 900)
Test.assert_equals(sum_polygon(8), 1080)
Test.assert_equals(sum_polygon(9), 1260)
Test.assert_equals(sum_polygon(10), 1440)
Test.assert_equals(sum_polygon(1000), 179640)
print('t_sec = {:.6f}'.format(perf_counter() - tic))